.travis/publish_multiarch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
copyright header is missing but it also don't pass shellcheck.net

Reply:
can we move this folder to .ci/ folder (as we have a generic folder)

Reply:
Will add the Copyright header and make necessary changes. We can include all variables in double quotes to avoid shell check error except $AMEND.If we add $AMEND in quotes we get the below error

```
$ docker manifest create "${REGISTRY}/${image}:latest-${TRAVIS_TAG}" "${AMEND}"
invalid reference format
```

To avoid shellcheck in case of $AMEND, we can include `# shellcheck disable=SC2086` in publish_multiarch.sh
Let us know your thoughts on the same

Reply:
could you just use eval at the beginning of the line ?
```
$ eval docker manifest create "${REGISTRY}/${image}:latest-${TRAVIS_TAG}" "${AMEND}"
```